.travis.yml

```diff
@@ -5,6 +5,8 @@ env:
 before_install:
 - "export DISPLAY=:99.0"
 - "sh -e /etc/init.d/xvfb start"
+- "export PLUGIN_VERSION=`grep '<version>' pom.xml -m 1 | cut -d'>' -f 2 | cut -d'<' -f 1`"
```

Comment:
I'm not able to review this kind of grep expressions

Reply:
it basically greps the version from the pom xml.